cli/multisig.go

```diff
@@ -881,6 +881,40 @@ var msigAddProposeCmd = &cli.Command{
 			from = defaddr
 		}
 
+		store := adt.WrapStore(ctx, cbor.NewCborStore(blockstore.NewAPIBlockstore(api)))
+
+		head, err := api.ChainHead(ctx)
+		if err != nil {
+			return err
+		}
+
+		act, err := api.StateGetActor(ctx, msig, head.Key())
+		if err != nil {
+			return err
+		}
+
+		mstate, err := multisig.Load(store, act)
+		if err != nil {
+			return err
+		}
+
+		signers, err := mstate.Signers()
+		if err != nil {
+			return err
+		}
+
+		for _, s := range signers {
+			signerActor, err := api.StateAccountKey(ctx, s, types.EmptyTSK)
```

Comment:
This won't work with actor addresses (e.g. a different multisig (f2 / f0 actor addresses))

It might be better to call `StateLookupID` on both the address that the user is trying to add, and the signer addresses.

Reply:
It does exist. It's been fixed